### ISSUE TITLE
Publish issue on GAE instance

### ISSUE THREAD

**Issue by kkrawczyk123** [opendatakit/aggregate#265]
#### Software and hardware versions 
Aggregate v1.6x, Java vx.x.x, ...

#### Problem description
When I try to Publish submissions to Google Spreadsheet or Fusion Tables, after accepting email address I can see an error message: 
![screenshot from 2018-06-26 10-16-56](https://user-images.githubusercontent.com/33342488/41898773-1cf57af8-792b-11e8-92c6-79d523516770.png)

#### Steps to reproduce the problem
1. I pulled code from github
2. I generated installers according to https://github.com/opendatakit/aggregate/blob/master/docs/installer-generation.md 
I used commands: ./gradlew clean build -x test -PwarMode=installer
and then: ./gradlew installerBuild
I used buildWar.xml to generate installers using installBuilder.
I used the same installer generation process in the last release of aggregate 1.5 with no problems.
3. Installed gae in compliance with https://docs.opendatakit.org/aggregate-app-engine/ 
I didn't notice any suspicious errors and installation succeeded. 
4. I opened a browser and I was able to run my aggregate. 
5. Then I configured API according to https://docs.opendatakit.org/oauth2-service/
6. and finally tried to publish submissions to Google Spreadsheet and Fusion Tables.

#### Expected behavior
I should definitely be able to publish submissions to Google Spreadsheet and Fusion Tables. 

#### Other information 
- Publishing to JSON server works well - the result is visible in Form Management/Published Data tab

- The issue doesn't occur on my tomcat instances. 

- The same API configuration works well on tomcat instance

- I am also able to visualize submissions on google map on gae instances. 

- I am able to use the same google accounts for publishing when using tomcat instance

- Issue occurs on all my gae instances no matter of operating system (tried installation on Ubuntu, MacOS and Windows) and java version (tried java 8 and 9 on all three systems)

**Comment by lognaturel** [comment]
Can you try going back to a previous release and see if you still get the same problem?

**Comment by lognaturel** [comment]
Report from user trying out the beta: https://forum.opendatakit.org/t/problem-publishing-from-odk-aggregate-to-google-spreadsheets/13743

**Comment by kkrawczyk123** [comment]
@lognaturel I am sure that we didn't have that problem in the previous release as it's a big one, the status of cases is testing plan connected with publishing is pass and I still have gae instances from previous release and publishing works well.
- I also checked api configuration of the instance when issue is occurring on last release's aggregate 1.5 and it works well.
- and set new gae instance using the installer from the forum - I still can reproduce that even if I don't know what exactly influent on it

**Comment by ggalmazor** [comment]
I think @kkrawczyk123 and I have cracked the issue. With the change of the engine to Java8, the default url stream handler requires a billing account (which I was using). I could reproduce the error by disabling the billing account.

We have learned that we can set another url stream handler that will work with both free and billing accounts. I have been able to fix the error by deploying a fixed version that uses the other url stream handler. 

Users having this problem should update their GAE instances using a new installer we need to provide. As far as I know, it's not possible to change `appengine-web.xml` files manually using the Google Cloud Console.
